test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommand_PrunePackageReference.cs

```diff
@@ -1386,19 +1396,25 @@ public void AnalyzePruningResults_WithMultiTargetingProjectAndVariousCasings_War
 
             var projectSpec = ProjectTestHelpers.GetPackageSpecWithProjectNameAndSpec("Project1", Path.GetTempPath(), rootProject);
             var testLogger = new TestLogger();
+            var testEvent = new TelemetryEvent("dummyEvent");
 
-            RestoreCommand.AnalyzePruningResults(projectSpec, testLogger);
+            RestoreCommand.AnalyzePruningResults(projectSpec, testEvent, testLogger);
 
             if (shouldWarn)
             {
                 testLogger.WarningMessages.Should().HaveCount(1);
                 var restoreLogMessage = (RestoreLogMessage)testLogger.LogMessages.Single();
                 restoreLogMessage.Code.Should().Be(NuGetLogCode.NU1510);
                 restoreLogMessage.LibraryId.Should().Be("a");
+                testEvent["Pruning.RemovablePackages.Count"].Should().Be(1);
```

Comment:
This and other places apply to my previous nit about the conditional.

Reply:
Will address as well.